images/router/f5/Dockerfile

```diff
@@ -0,0 +1,11 @@
+#
+# This is the HAProxy router for OpenShift Origin.
+#
+# The standard name for this image is openshift/origin-f5-router
+#
+FROM openshift/origin-base
+
+ADD bin/openshift /usr/bin/openshift
+RUN ln -s /usr/bin/openshift /usr/bin/openshift-router
+
+ENTRYPOINT ["/usr/bin/openshift-router", "--type=f5"]
```

Comment:
I'm referring to passing of the `--type`, the haproxy entrypoint now needs to pass `--type=template` if I understand correctly